config/end2end.php

```diff
@@ -1,10 +0,0 @@
-<?php
-
-return [
-    'api_url' => 'http://end2end--gateway.elife.internal',
```

Comment:
I just moved it to the front controller for the moment, but I agree that they shouldn't be hard-coded there.